package.json

```
@@ -12,7 +12,8 @@
     "lint": "npx eslint .",
     "start": "cds run",
     "test": "jest",
-    "test:mocha": "npx mocha --timeout 20000"
+    "test:mocha": "npx mocha --timeout 20000",
+    "watch-travel-analytics": "cds watch --open travel_analytics/webapp/index.html?sap-ui-xx-viewCache=false"
```

```suggestion
    "travel-analytics": "cds watch --open travel_analytics/webapp/index.html?sap-ui-xx-viewCache=false"
```

A command like `npm run travel-analytics` is linguistically nicer than  `npm run watch-travel-analytics` :)